### PR TITLE
chore: modify license notation

### PR DESCRIPTION
**Pull request by Pangssu**

### CODE REVIEW

**Comment by tynes** [comment]
Thank you!

File: packages/hardhat-deploy-config/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 (The MIT License)
 
-Copyright (c) 2023 Lightscale Inc.
+Copyright 2020-2023 Optimism
```
Comment (0xHansLee) [inline packages/hardhat-deploy-config/LICENSE:3]: We may need to also add MIT License of Lightscale as well as Optimism here, right?

Comment (seolaoh) [reply packages/hardhat-deploy-config/LICENSE:3]: Then I wonder if we don't need to change the `author` of `packages/hardhat-deploy-config/package.json`.

Comment (Pangssu) [reply packages/hardhat-deploy-config/LICENSE:3]: I also added to this package.

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by 0xHansLee — APPROVED** [review]
lgtm

**Review by kangsorang — APPROVED** [review]
LGTM

**Pull request merged**